__tests__/extensions/Relude_Extensions_Ord_test.re

```diff
@@ -1,6 +1,6 @@
 open Jest;
 open Expect;
-open Relude.Globals;
+open !Relude.Globals;
```

Comment:
I wonder if this should be `open! Relude.Globals` instead of `open !Relude.Globals`, but if it works this way, I'm equal parts confused an indifferent. 😁

Reply:
This is probably copy pasta. will fix